main.cpp

```diff
@@ -1,11 +1,14 @@
 #include <QApplication>
+#include <QDebug>
+#include <QDir>
 #include <QQmlApplicationEngine>
 
 int main(int argc, char *argv[])
 {
     QApplication app(argc, argv);
-
     QQmlApplicationEngine engine;
+
+    engine.addImportPath("qrc:/");
```

Comment:
I think there is no need for that, as well as there is no need to include QML material modules in qrc, since they are automatically handled by Qt for Android's packager.